frontend/src/metabase/common/utils/format-row-count.ts

```diff
@@ -0,0 +1,11 @@
+import { msgid, ngettext } from "ttag";
+
+import type { NumberFormatter } from "../hooks/use-number-formatter";
+
+export const formatRowCount = (
+  count: number,
+  formatNumber: NumberFormatter,
+) => {
+  const countString = formatNumber(count);
+  return ngettext(msgid`${countString} row`, `${countString} rows`, count);
```

Comment:
Just curious, how does using ngettext with msgid differ from using c` with interpolated strings? I usually do `c("{0} is the number of rows").jt`. I guess the difference is that this emits a plain JS string, while jt emits a ReactNode.

Reply:
> this emits a plain JS string, while jt emits a ReactNode

That, and also `msgid` is adapted for interpolated/complex expressions, while `t` and `jt` work for simple, direct translations.

Reply:
Thanks! I didn't realize `ngettext` existed so I _may_ have abused `c` in the past 😅